### PR TITLE
Spark 3.2

### PR DESCRIPTION
**Pull request by ingolfured**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The artifacts are published to [bintray](https://bintray.com/linkedin/maven/spar
 - Version 0.1.x targets Spark 2.3 and Scala 2.11
 - Version 0.2.x targets Spark 2.4 and both Scala 2.11 and 2.12
 - Version 0.3.x targets Spark 3.0 and Scala 2.12
+- Version 0.4.x targets Spark 3.2 and Scala 2.12
```
Comment (fliu-123) [inline README.md:13]: Is it possible to allow for scala 2.13 as well?

Comment (ingolfured) [reply README.md:13]: Yeah, I think it actually needs two different builds (they are not compatible in my local mvn builds). However, I think travis isn't being picked up anymore (needs a paid account?). So probably worth to migrate to circleCI or some free alternative.

Comment (ingolfured) [reply README.md:13]: Any chance you can make me an admin so I can set it up?

Comment (fliu-123) [reply README.md:13]: I am not part of the owner group.

**Comment by junshi15** [comment]
Hey @ingolfured thanks for the pull request. Let me take a look and merge it this weekend.

This is a company repo, it may be difficult to have external contributor to be an admin.

What are better alternatives to travis? Some folks mentioned Jenkins or circleCI. Let me know what you prefer, I will try to make the switch.

**Comment by junshi15** [comment]
@ingolfured v0.4.0 has been published. Thanks for your contribution.
In my limited testing, v0.3.4 (Spark 3.0 + Scala 2.12) seems works for Spark 3.2 as well.

**Pull request merged**